modules/logging-identity-federation.adoc

```diff
@@ -4,18 +4,22 @@
 :_mod-docs-content-type: PROCEDURE
 [id="logging-identity-federation_{context}"]
 = Workload identity federation
+
 Workload identity federation enables authentication to cloud-based log stores using short-lived tokens.
 
 .Prerequisites
 * {product-title} 4.14 and later
 * {logging-uc} 5.9 and later
 
 .Procedure
-* If you use the {product-title} web console to install the {loki-op}, clusters that use short-lived tokens are automatically detected. You are prompted to create roles and supply the data required for the {loki-op} to create a `CredentialsRequest` object, which populates a secret.
 
-* If you use the {oc-first} to install the {loki-op}, you must manually create a subscription object using the appropriate template for your storage provider, as shown in the following examples. This authentication strategy is only supported for the storage providers indicated.
+* Use one of the following options to enable authentication:
+
+** If you use the {product-title} web console to install the {loki-op}, clusters that use short-lived tokens are automatically detected. You are prompted to create roles and supply the data required for the {loki-op} to create a `CredentialsRequest` object, which populates a secret.
```

Comment:
Can we provide a YAML example to create a `CredentialsRequest` object for readers' reference?

Reply:
IIRC, Eng's take on this was that having an example of the CredentialsRequest was unnecessary and might be confusing, as it is generated for the user.